tests/installation/encrypted_volume/cancel_encrypted_volume.pm

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test module to cancel encrypted volume activation.
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my $encrypted_volume = $testapi::distri->get_encrypted_volume();
```

Comment:
let's use here our common approach to not include `use testapi` as it is not needed when we use variable `$testapi` and also make this one line: `$testapi::distri->get_encrypted_volume_activation()->do_not_activate()`.

Reply:
"use testapi" is needed for the $password. I know you like patterns and mathematics and tidy details and everything, but in this case, if we apply a strict pattern everywhere it would only make sense to use a similar approach as I am using because in a module that will call more than 2 commands, it wouldn't make sense to have this short of repetitions. I will not change it because it helps readability of the code a lot.

Reply:
you are not using $password for this test module and the library is not needed to be loaded, we can save that.
The less variable you declare in Perl, the better, is kind of the rule of the thumb, it is a very readable line, is not even long the one proposed, the plan is not touch the tests, so the idea is not to think about needing more than one line here, but let's leave it as it is.

Reply:
Done